feature/user/src/main/java/com/exalt/feature/user/components/ProfileHeader.kt

```diff
@@ -0,0 +1,85 @@
+package com.exalt.feature.user.components
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.exalt.core.domain.home.models.DomainModelFactory
+import com.exalt.core.ui.R
+import com.exalt.feature.user.enums.GenderConfig
+import com.exalt.feature.user.viewobjects.UserVO
+import com.google.accompanist.themeadapter.material3.Mdc3Theme
+
+@Composable
+fun ProfileHeader(userVO: UserVO, modifier: Modifier = Modifier) {
+    val imageSize = 100.dp
+    Box(
+        modifier
+            .height(imageSize)
+            .fillMaxWidth()
+    ) {
+
+        AsyncImage(
+            model = ImageRequest.Builder(LocalContext.current)
+                .data(userVO.profileBackground)
+                .crossfade(true)
+                .build(),
+            contentDescription = "profile background picture",
+            contentScale = ContentScale.FillBounds,
```

Comment:
```suggestion
            contentScale = ContentScale.Crop,
```

C'est plus joli, ça ne déforme pas l'image